internal/network/command_dns_forwarder.go

```diff
@@ -0,0 +1,119 @@
+package network
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	networkingdnsforwarderv1 "github.com/confluentinc/ccloud-sdk-go-v2-internal/networking-dnsforwarder/v1"
+
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type dnsForwarderHumanOut struct {
+	Id           string `human:"ID"`
+	Name         string `human:"Display Name,omitempty"`
+	Domains      string `human:"Domains,omitempty"`
+	DnsServerIps string `human:"DNS Server IPs"`
+	Environment  string `human:"Environment"`
+	Gateway      string `human:"Gateway"`
+	Phase        string `human:"Phase"`
+}
+
+type dnsForwarderSerializedOut struct {
+	Id           string   `serialized:"id"`
+	Name         string   `serialized:"display_name,omitempty"`
+	Domains      []string `serialized:"domains,omitempty"`
+	DnsServerIps []string `serialized:"dns_server_ips"`
+	Environment  string   `serialized:"environment"`
+	Gateway      string   `serialized:"gateway"`
+	Phase        string   `serialized:"phase"`
+}
+
+func (c *command) newDnsForwarderCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "dns-forwarder",
```

Comment:
We will be adding supports for managing `dns records` to the CLI for the Outbound PL project.
If we follow this convention, we will have `network dns-forwarder` and `network dns-record`.

Alternatively, we can have a sub-command `dns` under `network`
`network dns forwarder`
`network dns record`

@kabraham13 which one do you prefer?

Reply:
I would prefer a `dns` subcommand. 

So: 
`network dns forwarder`
`network dns record`